tests/drop_enum_test.js

```diff
@@ -1,35 +1,34 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
 require('./helper');
 
-var expect = require('chai').expect;
-var Sequelize = require('..');
-var DataTypes = Sequelize.DataTypes;
+const { expect } = require('chai');
+const { Sequelize, DataTypes } = require('../source');
 
-// Reason: after merging with other patches started failing locally only
-describe.skip('QueryInterface', () => {
+describe('QueryInterface', () => {
   beforeEach(function() {
-    this.sequelize.options.quoteIdenifiers = true;
+    this.sequelize.options.quoteIdentifiers = true;
     this.queryInterface = this.sequelize.getQueryInterface();
   });
 
-  afterEach(async function() {
-    this.queryInterface.dropTable('menus');
-  });
-
   describe('dropEnum', () => {
     beforeEach(async function() {
-     await this.queryInterface.createTable('menus',  {
-        structuretype: {
-          type: DataTypes.ENUM('menus', 'submenu', 'routine'),
-          allowNull: true
-        },
-        sequence: {
-          type: DataTypes.INTEGER,
-          allowNull: true
-        },
-        name: {
-          type: DataTypes.STRING,
-          allowNull: true
-        }
+      await this.queryInterface.createTable('menus', {
```

Comment:
Just curious, what's the reason behind this change? I use the sequelize equivalent tests for reference and they define the params the way we did before 

https://github.com/sequelize/sequelize/blob/main/test/integration/query-interface/dropEnum.test.js#L21

Reply:
Oh! This change is not important, it was just a refactor that I felt like doing. The behavior is unchanged, since `allowNull` is already `true` by default. I prefer shorter tests, so I like to omit defaults like these. However, I didn't notice this was based on an existing Sequelize test. Thanks for bringing it to my attention, I've just opened [a PR](https://github.com/sequelize/sequelize/pull/13084) on Sequelize itself to perform the same refactor there.